### PR TITLE
magento/devdocs#6310: Clarify steps in Migration Guide

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/devdocs/issues/6310.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-settings.html

## Links to Magento source code

-  ...

Thanks!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/migration/migration-migrate-settings.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ According to our data migration [order]({{ page.baseurl }}/migration/migration-m
 
 ## Before you start: routine preparations
 
-1. Log in to Magento server as [the file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
+1. Log in to Magento server (where you have M1 and M2 installed) as [the file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
```
Comment (rogyar) [inline src/guides/v2.3/migration/migration-migrate-settings.md]: > (where you have M1 and M2 installed) 

I'm not sure this assumption is 100% correct. You may not have Magento 1 installed on the server where you are executing the data migration. Your Magento 2 server may only have Magento 1 database locally or the access details for the remote Magento 1 database. 

So, here we are rather talking about 

> Log in to the server with your Magento 2 instance

File: src/guides/v2.3/migration/migration-migrate-settings.md [hunk 2]
```diff
@@ -48,17 +48,27 @@ The `Migration completed` message is displayed after the settings are transferre
 
 You may ignore, rename or change the system configurations when migrating settings. For this, specify your custom rules in the `settings.xml` file.
 
-1. Log in to your Magento server as, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/apache-user.html).
+1. Log in to your Magento server (where you have M1 and M2 installed) as, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/apache-user.html).
```
Comment (rogyar) [inline src/guides/v2.3/migration/migration-migrate-settings.md]: Please, check my comment above

File: src/guides/v2.3/migration/migration-migrate-settings.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.3/migration/migration-migrate-settings.md]: Hi, @rogyar !

I agree with your notice. 

**fixed**

Could you please verify?

Thanks

File: src/guides/v2.3/migration/migration-migrate-settings.md (hunk 2 above)
Comment (atwixfirster) [reply src/guides/v2.3/migration/migration-migrate-settings.md]: **fixed**

Thanks

**Comment by rogyar** [comment]
Thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**